mne/io/boxy/tests/test_boxy.py

```diff
@@ -0,0 +1,205 @@
+# Authors: Kyle Mathewson, Jonathan Kuziek <kuziek@ualberta.ca>
+#
+# License: BSD (3-clause)
+
+import os
+
+import numpy as np
+from numpy.testing import assert_allclose, assert_array_equal
+import scipy.io as spio
+
+import mne
+from mne.datasets.testing import data_path, requires_testing_data
+
+
+@requires_testing_data
+def test_boxy_load():
+    """Test reading BOXY files."""
+    # Determine to which decimal place we will compare.
+    thresh = 1e-10
+
+    # Load AC, DC, and Phase data.
+    boxy_file = os.path.join(data_path(download=False),
```

Comment:
This data has no annotations correct? Then can we test that no annotations are returned?

Reply:
sure thing, I can add this